previous-version/README.md

```diff
@@ -0,0 +1,40 @@
+# previous-version
+
+Given a release version identifies release type and a corresponding previous release version
+
+## Usage
+
+This GHA can be used in any repository, but requires it to be fetched with full history,
+so that existing tags can be analyzed. Otherwise, results may be incorrect.
+
+### Inputs
+| Input name | Description                            |
+|------------|----------------------------------------|
+| version    | current release version (e.g. 8.8.0)   |
+| verbose    | provide verbose output (true \| false) |
```

Comment:
let's add that default value for `verbose` is `false` and that `version` is required.